packages/ui/src/components/text.tsx

```diff
@@ -41,12 +56,26 @@ const textVariants = cva('text-base', {
       semibold: 'font-semibold',
       bold: 'font-bold'
     },
+    // TODO: Color system requires refactoring
+    // Problems:
+    // 1. Abstract naming (primary, secondary) doesn't indicate actual colors
```

Comment:
this is intentional. there's a bunch of discussion behind this decision documented here: https://harness.atlassian.net/wiki/spaces/~5c57a22c0f8313373e56a974/pages/21932900533/Design+System+Pipeline+Spec

Maybe let's discuss this before we change this?

Reply:
Not sure if you can access that doc ^.

But here's a similar public third-party article: https://medium.com/eightshapes-llc/naming-tokens-in-design-systems-9e86c7444676